framework/web/Request.php

```diff
@@ -1444,8 +1419,8 @@ public function validateCsrfToken($token = null)
     /**
      * Validates CSRF token
      *
-     * @param string $token
-     * @param string $trueToken
+     * @param string $token The masked client-supplied token.
```

Comment:
I am fine with whatever small changes, discussing those is not a valuable way to spend my time.
But it is also a waste of my time if they keep coming, and even more if you don't even agree internally ;-)


If there is concensus on the new name ping me and I'll rename it.